src/components/List.js

```diff
@@ -116,15 +108,11 @@ const List = ({ token }) => {
 
   const backgroundColor = (item) => {
     const daysUntilNextPurchase = getDaysUntilNextPurchase(item);
-    if (isInactive(item)) {
-      return 'inactive';
-    } else if (daysUntilNextPurchase < 7) {
-      return 'soon';
-    } else if (daysUntilNextPurchase >= 7 && daysUntilNextPurchase < 30) {
+    if (isInactive(item)) return 'inactive';
+    if (daysUntilNextPurchase < 7) return 'soon';
+    if (daysUntilNextPurchase >= 7 && daysUntilNextPurchase < 30)
       return 'kind-of-soon';
```

Comment:
It might be nice to inline this return statement too if all the others are like that, but maybe this is prettier doing it for us, in which case I'd defer to the tool

Reply:
@chrisdopuch That's prettier doing it's thing 😄